diff --git a/docroot/.htaccess b/docroot/.htaccess
index d158782..cf2e2b9 100644
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -145,6 +145,33 @@ AddEncoding gzip svgz
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]
 
+  # Exclude Acquia domains
+  RewriteCond %{HTTP_HOST} !\.acquia-sites\.com [NC]
+  # Exclude Acquia Cloud IDE preview sites
+  RewriteCond %{HTTP_HOST} !\.web\.ahdev\.cloud [NC]
+  # Ensure non www.
+  RewriteCond %{HTTP_HOST} ^www\.(.*)$ [NC]
+  RewriteRule ^(.*)$ https://%1/$1 [R=301,L]
+  #   # Ensure non www.
+ https
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+  # Redirect to beta domain
+  RewriteCond %{HTTP_HOST} ^www.digitalsocialcare.co.uk$ [NC,OR]
+  RewriteCond %{HTTP_HOST} ^digitalsocialcare.co.uk$ [NC,OR]
+  RewriteCond %{HTTP_HOST} ^www.digitisingsocialcare.co.uk$ [NC,OR]
+  RewriteCond %{HTTP_HOST} ^digitisingsocialcare.co.uk$ [NC]
+  RewriteRule ^(.*)$ https://beta.digitisingsocialcare.co.uk/$1 [R=301,L]
+  # Redirect to api-data subdomain
+  RewriteCond %{HTTP_HOST} ^api-data.digitalsocialcare.co.uk$ [NC]
+  RewriteRule ^(.*)$ https://api-data.digitisingsocialcare.co.uk/$1 [R=301,L]
+  # Redirect to data subdomain
+  RewriteCond %{HTTP_HOST} ^data.digitalsocialcare.co.uk$ [NC]
+  RewriteRule ^(.*)$ https://data.digitisingsocialcare.co.uk/$1 [R=301,L]
+  # Redirect to develop subdomain
+  RewriteCond %{HTTP_HOST} ^develop.digitalsocialcare.co.uk$ [NC]
+  RewriteRule ^(.*)$ https://develop.digitisingsocialcare.co.uk/$1 [R=301,L]
+
   # Rules to correctly serve gzip compressed CSS and JS files.
   # Requires both mod_rewrite and mod_headers to be enabled.
   <IfModule mod_headers.c>
