diff --git a/docroot/.htaccess b/docroot/.htaccess
index 4031da4..7e2f675 100644
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -145,6 +145,21 @@ AddEncoding gzip svgz
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]

+  # exclude Acquia domains
+  RewriteCond %{HTTP_HOST} !\.acquia-sites\.com [NC]
+  # exclude Acquia Cloud IDE preview sites
+  RewriteCond %{HTTP_HOST} !\.web\.ahdev\.cloud [NC]
+  # Redirect www requests to beta subdomain
+  RewriteCond %{HTTP_HOST} ^digitalsocialcare\.co\.uk [NC,OR]
+  RewriteCond %{HTTP_HOST} ^www\.digitalsocialcare\.co\.uk [NC]
+  RewriteRule ^(.*)$ https://beta.digitisingsocialcare.co.uk/$1 [L,R=301]
+  # Preserve subdomains if initially requested
+  RewriteCond %{HTTP_HOST} !^beta\.digitisingsocialcare\.co\.uk [NC,OR]
+  RewriteCond %{HTTP_HOST} ^([^.]+)\.digitalsocialcare\.co\.uk [NC]
+  RewriteRule ^(.*)$ https://%1.digitisingsocialcare.co.uk/$1 [L,R=301]
+
   # Rules to correctly serve gzip compressed CSS and JS files.
   # Requires both mod_rewrite and mod_headers to be enabled.
   <IfModule mod_headers.c>
