diff --git a/docroot/.htaccess b/docroot/.htaccess
index 4031da4..30a282c 100644
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -113,6 +113,19 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
+  # exclude Acquia domains
+  RewriteCond %{HTTP_HOST} !\.acquia-sites\.com [NC]
+  # exclude Acquia Cloud IDE preview sites
+  RewriteCond %{HTTP_HOST} !\.web\.ahdev\.cloud [NC]
+  # Redirect www requests to beta subdomain
+  RewriteCond %{HTTP_HOST} ^digitalsocialcare\.co\.uk [NC,OR]
+  RewriteCond %{HTTP_HOST} ^www\.digitalsocialcare\.co\.uk [NC]
+  RewriteRule ^(.*)$ https://beta.digitisingsocialcare.co.uk/$1 [L,R=301]
+  # Preserve subdomains if initially requested
+  RewriteCond %{HTTP_HOST} !^beta\.digitisingsocialcare\.co\.uk [NC,OR]
+  RewriteCond %{HTTP_HOST} ^([^.]+)\.digitalsocialcare\.co\.uk [NC]
+  RewriteRule ^(.*)$ https://%1.digitisingsocialcare.co.uk/$1 [L,R=301]
+
   # Redirect common PHP files to their new locations.
   RewriteCond %{REQUEST_URI} ^(.*)?/(install\.php) [OR]
   RewriteCond %{REQUEST_URI} ^(.*)?/(rebuild\.php)
