diff --git a/docroot/.htaccess b/docroot/.htaccess
index 1575c51..4e1da18 100644
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -152,6 +152,20 @@ AddEncoding gzip svgz
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]
 
+  # exclude Acquia domains
+  RewriteCond %{HTTP_HOST} !\.acquia-sites\.com [NC]
+  # exclude Acquia Cloud IDE preview sites
+  RewriteCond %{HTTP_HOST} !\.web\.ahdev\.cloud [NC]
+  # ensure non www.
+  RewriteCond %{HTTP_HOST} ^www\.(.*)$ [NC]
+  RewriteRule ^(.*)$ https://%1/$1 [R=301,L]
+  # ensure https
+  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+  # redirect main domain to beta url
+  RewriteCond %{HTTP_HOST} ^dev.digitisingsocialcare.co.uk$ [NC]
+  RewriteRule ^ https://beta.digitisingsocialcare.co.uk/$1 [R=301,L]
+
   # Rules to correctly serve gzip compressed CSS and JS files.
   # Requires both mod_rewrite and mod_headers to be enabled.
   <IfModule mod_headers.c>
